### DISCUSSION TITLE
アイコンは変更できません

### DISCUSSION THREAD

**Discussion by Lunar0922** [misskey-dev/misskey#14299] [Q&A]
### 💡 Summary

アイコンは変更できません
![屏幕截图 2024-07-23 135524](https://github.com/user-attachments/assets/8bb43d26-d0a7-446e-a55d-0c932b009184)

### 🥰 Expected Behavior

顔は真っ白ではなく写真全体になるべきだ

### 🤬 Actual Behavior

アイコンは変更できません

### 💻 Frontend Environment

```markdown
* Model and OS of the device(s):Linux debian10
* Browser:Chrome 113.0.5672.126
* Server URL:agzx.top
* Misskey:2024.05
```

### 🛰 Backend Environment (for server admin)

```markdown
* Installation Method or Hosting Service:：docker compose
* Misskey:2024.05
* Node:20.1
* PostgreSQL:15.5
* Redis:7.18
* OS and Architectu debian10
```

### Do you want to address this bug yourself?

- [ ] Yes, I will patch the bug myself and send a pull request

**Comment by kakkokari-gtyih** [comment]
Please fill in the `Steps to Reproduce` as we have no idea what did you do to the profile picture.

**Comment by zyoshoka** [comment]
Looks like `url` is not set correctly in your `.config/default.yml` file.

**Comment by Lunar0922** [comment]
どのようにそれを修正するのですか？

**Comment by samunohito** [comment]
これは、私が過去に経験した事から考えた仮説です。

まず、あなたのMisskeyの `.config/default.yml` を開き、 `url` の値を確認してください。
そして、あなたがMisskeyを公開しているオリジンと `url` の値が同一であるかを確認します。

もし同一ではない場合、Misskeyを公開しているオリジンと合わせてください。
（Misskeyを公開しているURLが `https://agzx.top/` であると仮定すると、 `url` の値は `https://agzx.top/` となるはずです）

`url` はアップロードされたファイルにアクセスするためのURLを生成するだけでなく、様々な箇所で使用されます。
そのため、想定されていない値を設定するとMisskeyが正しく動作しない可能性が高くなります。

**Comment by kakkokari-gtyih** [comment]
Converting to discussion as thi does not appear to be a bug
